README.md

```diff
@@ -43,23 +43,29 @@ npm install
 npm run build:prod:watch
 ```
 
-You can also manually adjusting your API endpoint settings in `webpack.config.js` by altering the `targets` within `proxy`. For example:
+Or you can set the ENV variable to `production` and run webpack manually.
+
+```
+cross-env NODE_ENV=production ENV=production webpack
+```
+
```

Comment:
And this can probably be removed since I missed that we have more npm run commands.
```suggestion
```